Automattic/jetpack
# Widgets: Adds support for Simple Payment Buttons as Widgets

**DO NOT MERGE - THIS A FEATURE BRANCH**

Depends on:

- Simple Payments: adds plan support check for Jetpack and WordPress.com (#9650 and D14005-code)

#### Changes proposed in this Pull Request:

Adds basic support on the Customizer to add Simple Payment Buttons as Sidebar Widgets.

| Widget List | Widget Customizer Form & Preview |
| - | - |
| ![screen shot 2018-05-18 at 12 14 37](https://user-images.githubusercontent.com/233601/40242922-5af9c0ac-5a95-11e8-9929-945e2c4e7e21.png) | ![screen shot 2018-05-18 at 12 15 34](https://user-images.githubusercontent.com/233601/40242925-5fb172b6-5a95-11e8-889b-4b5763f08170.png) |

#### Testing instructions:

Pre-requisites:
- apply this branch on a Jetpack site with an active Business Plan subscription and a Theme with at least one widget area
- to create one or more Payment Buttons on the Post/Page editor

then you'll need to:

- open the Customizer on your sandboxed Business Site
- go to Widgets and select a Widget Area
- click on Add Widget
- search on the widget panel for Simple Payment
- select the Simple Payment widget

The Simple Payment Customizer should list the available Payment Buttons, and the Customizer Preview should show the item selected by default. All changes should update live on the preview window and the site should only be updated upon publishing.

Known Issues

- sometimes the live update removes the PayPal button.
- if there are no Payment Buttons, it just doesn't show a drop down.
- if the selected Payment Button is deleted, the widget just shows the title and empty content.

#### Proposed changelog entry for your changes:
